internal/provider/vsphere/installConfig.go

```diff
@@ -18,18 +19,26 @@ func setPlatformValues(platform *installcfg.VsphereInstallConfigPlatform) {
 	platform.Datacenter = PhDatacenter
 }
 
-func (p vsphereProvider) AddPlatformToInstallConfig(
-	cfg *installcfg.InstallerConfigBaremetal, cluster *common.Cluster) error {
-	if len(cluster.APIVip) == 0 {
+func (p vsphereProvider) AddPlatformToInstallConfig(cfg *installcfg.InstallerConfigBaremetal, cluster *common.Cluster) error {
+	if len(cluster.APIVip) == 0 && !swag.BoolValue(cluster.UserManagedNetworking) {
 		return errors.New("invalid cluster parameters, APIVip must be provided")
 	}
-	if len(cluster.IngressVip) == 0 {
-		return errors.New("invalid cluster parameters, IngressVip must be provided")
-	}
-	vsPlatform := &installcfg.VsphereInstallConfigPlatform{
-		APIVIP:     cluster.APIVip,
-		IngressVIP: cluster.IngressVip,
+
+	vsPlatform := &installcfg.VsphereInstallConfigPlatform{}
+
+	if !swag.BoolValue(cluster.UserManagedNetworking) {
```

Comment:
Please add a UT for this case

Reply:
https://github.com/openshift/assisted-service/pull/4384#issuecomment-1243648980

Reply:
But we do have UT for this case .. every install-config none-platform test is checking that